src/twisted/internet/test/test_tcp.py

```diff
@@ -56,12 +56,16 @@
     from twisted.internet.ssl import ClientContextFactory
     useSSL = True
 
+s = None
 try:
-    socket.socket(socket.AF_INET6, socket.SOCK_STREAM).close()
+    s = socket.socket(socket.AF_INET6, socket.SOCK_STREAM)
+    s.bind(('::1', 0))
 except socket.error as e:
     ipv6Skip = str(e)
 else:
     ipv6Skip = None
+if s is not None:
+    s.close()
```

Comment:
We're going to not do that this time around because we'd rather add this check as productized code, so @markrwilliams is going to do that while we merge this to unblock you.